lib/backend/test/suite.go

```diff
@@ -578,7 +587,7 @@ func requireEvent(t *testing.T, watcher backend.Watcher, eventType types.OpType,
 		require.FailNow(t, "Watcher has unexpectedly closed.")
 
 	case <-time.After(timeout):
-		require.FailNow(t, "Timed out after %v waiting for event %v", timeout, eventType.String())
+		require.FailNowf(t, "Timed out", "Timed out after %v waiting for event %v", timeout.String(), eventType.String())
```

Comment:
nit:
```suggestion
		require.FailNowf(t, "Timed out after %v waiting for event %v", timeout.String(), eventType.String())
```

Reply:
That will give errors like:
```
    firestorebk_test.go:158: 
                Error Trace:    firestorebk_test.go:158
                Error:          Timed out after %v waiting for event %v
                Test:           TestFail
                Messages:       10s%!(EXTRA string=some event)
--- FAIL: TestFail (0.00s)
```

My change makes it give:
```
    firestorebk_test.go:158: 
                Error Trace:    firestorebk_test.go:158
                Error:          Timed out
                Test:           TestFail
                Messages:       Timed out after 10s waiting for event some event
--- FAIL: TestFail (0.00s)
```